File: src/main/java/com/linelife/demo/model/constant/Pulse.java
package com.linelife.demo.model.constant;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Data
@Table(name = "pulse")
public class Pulse {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    @JsonIgnore
    private Long id;
    @Column(name = "user_id")
    @JsonProperty(value = "user_id")
    private Long userId;
    @Column(name = "date_time")
    @JsonProperty(value = "date_time")
    private LocalDateTime dateTime;
    @Column(name = "pulse")
    private Integer pulse;
}

File: app/src/main/java/com/example/lifeline/data/Pulse.kt
package com.example.lifeline.data

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

data class Pulse(
        @SerializedName("user_id")
        var userId:Long,
        @SerializedName("date_time")
        var dateTime:LocalDateTime,
        @SerializedName("pulse")
        var value:Int
)